build.gradle

```diff
@@ -23,7 +23,9 @@ ext {
   organizationName = "Adaptris Ltd"
   slf4jVersion = '2.0.3'
   mockitoVersion = '4.9.0'
-
+  
+  interlokCommonsOfflineJavadocDir = project.rootDir.getCanonicalPath() + "/interlok-common/build/docs/javadoc"
```

Comment:
I don't think you need this line `interlokCommonsOfflineJavadocDir = project.rootDir.getCanonicalPath() + "/interlok-common/build/docs/javadoc"`
And maybe not the one after for lombok, I'm not sure if this project is using lombok.

Reply:
same as the comment above, saw it in interlok-core and thought it was one of the required changes to the build file for junit 5 and java 17.